src/branching.md

```diff
@@ -0,0 +1,53 @@
+# Branching strategy
+
+- [main](https://github.com/microsoft/cloud-hubs/tree/main) includes the latest stable release.
+- [dev](https://github.com/microsoft/cloud-hubs/tree/dev) includes the latest changes that will go into the next release.
+- Feature branches (`features/<feature-name>`) are used for any in-progress features that are not yet ready for release.
+- Personal branches (`<your-github-account>/<feature-name>`) are intended for a single developer and typically not shared. Use these for small changes that can easily be integrated into the next release.
+
+```mermaid
```

Comment:
Use the "rich diff" to see what this looks like:
![image](https://user-images.githubusercontent.com/399533/219975662-4addc374-e049-4da0-9054-d2d7724c05f5.png)